documentation/docs/single_server_mode.md

```diff
@@ -0,0 +1,78 @@
+
+The single server mode uses an embedded SQLite database stored on a filesystem and therefore does not require access to an external database.
+
+Although it does not provide high availability through redundancy, this configuration is suitable for production and serving millions of cryptographic objects.  The server will concurrently serve requests on as many threads as available cores to the docker container.
+
+This configuration also supports client-secret encrypted databases, a secure way to store cryptographic objects since database keys are provisioned on every request, and no database key is stored server side. To offer a fully secure solution suitable for deployment in a zero-trust environment such as the cloud, TLS must be enabled on the server, and the memory of the KMS server must also be protected during operation by running the server inside an enclave. Ask Cosmian for details
```

Comment:
```suggestion
This configuration also supports client-secret encrypted databases, a secure way to store cryptographic objects since database keys are provisioned on every request, and no database key is stored server side. To offer a fully secure solution suitable for deployment in a zero-trust environment such as the cloud, TLS must be enabled on the server, and the memory of the KMS server must also be protected during operation by running the server inside an enclave. Ask Cosmian for details.
```